pkg/segment/sortindex/sortindex.go

```diff
@@ -43,6 +43,10 @@ func getFilename(segkey string, cname string) string {
 	return filepath.Join(segkey, cname+".srt") // srt means "sort", not an acronym
 }
 
+func getTempFilename(segkey string, cname string) string {
+	return getFilename(segkey, cname) + ".tmp"
```

Comment:
Nit: shouldn't this be blah.srt.tmp instead of blah.tmp

Reply:
It will be. `getFilename()` will return `blah.srt`